lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -384,24 +245,6 @@ module Resources
             }
           ],
           'text': 'Default Guarantor'
-        },
-        {
-          'id': 'CI-97982588-1-0',
-          'coding': [
-            {
-              'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/40',
-              'code': '670847',
-              'display': 'Child',
-              'userSelected': true
-            },
-            {
-              'system': 'http://terminology.hl7.org/CodeSystem/v3-RoleCode',
-              'code': 'CHILD',
-              'display': 'child',
-              'userSelected': false
-            }
-          ],
```

Comment:
This relationship shouldn't be removed yet (unless it was just a different example from the db?), since we still return this today.

Reply:
added back here https://github.com/cerner/fhir.cerner.com/pull/624/commits/906c8894c70e01bd63fcd1eae7377f2bd18849c5#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR248-R263